config.example.toml

```
@@ -756,6 +756,9 @@
 # Build compiler with the optimization enabled and -Zvalidate-mir, currently only for `std`
 #validate-mir-opts = 3
 
+# Configure `std` features used during bootstrap
+# std_features = ["panic_unwind"]
```

There shouldn't be any whitespace before the key; otherwise the configure script won't work.

```suggestion
#std_features = ["panic_unwind"]
```